src/main/java/com/team4/testingsystem/services/impl/TestsServiceImpl.java

```diff
@@ -69,6 +69,11 @@ public Test getById(long id) {
         return testsRepository.getByStatuses(statuses);
     }
 
+    @Override
+    public List<Test> getAllUnverifiedTestsByCoach(long coachId) {
+        return testsRepository.getAllByAssignedCoachAndStatus(coachId, Status.COMPLETED);
```

Comment:
in-verification?

Reply:
This page is for coaches to start test verification. The page displays all **unverified** tests, that were assigned to the current user.

Reply:
So, when the status is `in-verification`, the test shouldn't be displayed here)

Reply:
They should. The coach could leave a page and should have ability to continue the verification

Reply:
Oh, yeah. I'll rename the method.

Reply:
Oh, maybe I don't need to rename it :)